src/components/header.tsx

```diff
@@ -1,109 +1,237 @@
+            <ListItemText primary="About Us" />
+          </ListItemButton>
+        </ListItem>
+        <ListItem disablePadding>
+          <ListItemButton
+            component="a"
+            href="#sponsors"
+            onClick={handleDrawerToggle}
           >
-            here
-          </a>{" "}
-          to view the application guide on CVs and resumes.
-        </p>
-        <p>
-          Click{" "}
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href="https://docs.google.com/document/d/1mGjAsgTP7KIheWVDm-TZ5Hxxf8WYPfWemiqcZ40XCwo/edit?usp=sharing"
+            <ListItemText primary="Sponsors" />
+          </ListItemButton>
+        </ListItem>
+        <ListItem disablePadding>
+          <ListItemButton
+            component="a"
+            href="#mentors"
+            onClick={handleDrawerToggle}
           >
-            here
-          </a>{" "}
-          to view a list of FAQs for applicants.
-        </p>
-        <p>
-          Click{" "}
-          <a
+            <ListItemText primary="Mentors" />
+          </ListItemButton>
+        </ListItem>
+        <ListItem disablePadding>
+          <ListItemButton
+            component="a"
+            href="https://advisorysg.typeform.com/to/HQ8nWq2r#source=mentorsite"
             target="_blank"
             rel="noopener noreferrer"
-            href="https://drive.google.com/file/d/1aA3l_WQ1ehGO8UBKuRFLqj4URpQbAVzN/view?usp=sharing"
           >
-            here
-          </a>{" "}
-          to view a sample application response.
-        </p>
-        <p>
-          Click{" "}
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href="https://drive.google.com/file/d/19lUr2BchbYFAPMjEbN_M2mf1pyw-n7pj/view?usp=sharing"
+            <ListItemText primary="Apply Now" />
+          </ListItemButton>
+        </ListItem>
+      </List>
+      <Divider />
+    </div>
+  );
+  const container =
+    window !== undefined ? () => window().document.body : undefined;
+
+  return (
+    <Box sx={{ display: "flex" }}>
+      <AppBar style={{ background: "#f89b27" }} className="toolbar-component">
+        <Toolbar>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            edge="start"
+            onClick={handleDrawerToggle}
+            sx={{ mr: 2, display: { sm: "none" } }}
           >
-            here
-          </a>{" "}
-          to view a list of 48 industries to filter mentors by industry.
-        </p>
-      </div>
+            <MenuIcon />
+          </IconButton>
+          <img
+            className="nav-logo"
+            src={advisoryLogo}
+            alt="Advisory"
+            style={{ alignSelf: "center" }}
+          />
+          <a className="nav-back-to-main-site" href="https://advisory.sg">
+            &#x3c; back to main site
+          </a>
+        </Toolbar>
+      </AppBar>
```

Comment:
I do agree with you! Hmm but I felt that it would be better to have that top bar so that I could make it sticky and use it for both on small screens and large screens, without needing to write another separate header? Not sure if my understanding is right!